src/main/java/org/ibp/api/java/impl/middleware/common/validator/LocationValidator.java

```diff
@@ -229,16 +229,21 @@ public void validateCanBeDeleted(final Integer locationId) {
 	}
 
 	private void validateLocationNotDeletable(final LocationDTO locationDTO) {
-		if (LOCATIONS_NOT_DELETABLES.contains(locationDTO.getName()) || locationDTO.isDefaultLocation()) {
+		if (LOCATIONS_NOT_DELETABLES.contains(locationDTO.getName())) {
```

Comment:
@mderamos @mcrimi 
Does validating that the following locations can not be deleted still make sense : "Unspecified Location", "Default Seed Store", "Default Breeding Location"? 
With the new implementation seems that they can be removed if in any instance they are not used as default anymore

Reply:
Hi @clarysabel, I agree. Also setting the Unspecified Location as program favorite upon program creation seems unnecessary since the users are selecting default locations and setting the selected locations as favorites.

Reply:
Hi @mderamos, yes, that sounds reasonable to me. cc: @clarysabel